cmd/nodecmd/local.go

```diff
@@ -511,3 +508,8 @@ func localTrack(_ *cobra.Command, args []string) error {
 	ux.Logger.GreenCheckmarkToUser("%s successfully tracking %s", clusterName, blockchainName)
 	return nil
 }
+
+func notImplementedForLocal(clusterName string, what string) error {
+	ux.Logger.PrintToUser("Local cluster %s does not support %s cmd", logging.LightBlue.Wrap(clusterName), logging.LightBlue.Wrap(what))
```

Comment:
I would rather use "Unsupported cmd: %s is not supported by local clusters". A nit feel free to think on this
and find out a suitable variation

Reply:
agree. changed